poptorch_experimental_addons/_impl/collectives.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Graphcore Ltd. All rights reserved.
+
+import torch
```

Comment:
For consistency, `import torch as T` (or we can go back & change the sparsity ops, perhaps in a separate PR).

Reply:
I can do, although I think it is generally unconventional to alias `torch`. Doesn't appear in https://import-as.github.io/ for example.

Reply:
Agree - addressed in #4